### PR TITLE
azure-information-protection-schema.md

### PR DESCRIPTION
**Pull request by krishnacloud123**
Created .md page for AIP Schema for events (93 , 94 , 97). @lindalu-MSFT

### CODE REVIEW

File: azure-information-protection-schema.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+|UserKey| AdeleV@M365x23987777.OnMicrosoft.com| An alternative ID for the user identified in the UserId property. This property is populated with the passport unique ID (PUID) for events performed by users in SharePoint, OneDrive for Business, and Exchange.| 
+|Workload|Aip|Stores the workload type of the Office 365 service where the activity occurred.|
+|Version|1|Version of the Azure Information Protection client that performed the audit action|
+|Scope|1|xx|
+
+## AipProtectionAction
+### AIP protection events.
+This is event 95 - This event contains AIP protection events.
+
+## AipFileDeleted.
```
Comment (lindalu-MSFT) [inline azure-information-protection-schema.md:126]:
```suggestion
## AipFileDeleted
```

**Review by lindalu-MSFT — CHANGES_REQUESTED** [review]
Hi, I added the metadata at the top and bookmarks. There is no content for events 95 and 96, is that intentional or are they in the wrong locations in this topic? Let me know if you have questions.

**Comment by krishnacloud123** [comment]
> Hi, I added the metadata at the top and bookmarks. There is no content for events 95 and 96, is that intentional or are they in the wrong locations in this topic? Let me know if you have questions.

95 and 96 is WIP and looking to get it completed and update the document at the earliest.

**Comment by krishnacloud123** [comment]
Just wanted to check if there are any more pending items for one this – before the document can be published.

Krishna ***@***.***)
Cloud Solution Architect

**Comment by lindalu-MSFT** [comment]
Do you need a quick phone call to discuss this PR? I see lots of commits that are not in your branch and all the revisions I requested are still there. I’d be happy to discuss these changes with you and we can get this published.

**Review by lindalu-MSFT — APPROVED** [review]

**Pull request merged**